### ISSUE TITLE
Hello, bilateralfilter.py is missing in the code you gave, can you provide it？

### ISSUE THREAD

**Issue by jijihengheng** [rulixiang/wetr#3]
Hello, bilateralfilter.py is missing in the code you gave, can you provide it？

**Comment by rulixiang** [comment]
Hey, @jijihengheng , to use the `bilateralfilter.py`/regularized loss, you need to compile it on your platform. See Issue #1 for help.

**Comment by jijihengheng** [comment]
Hello,@rulixiang! 
When I run the program, I get the following warnings and hints：
(1)INFO:Added key: store_based_barrier_key:1 to store for rank: 0
(2) Warning: Grad strides do not match bucket view strides. This may indicate grad was not created according to the gradient layout contract, or that the param's strides changed since DDP was construct
ed.  This is not an error, but may impair performance.
grad.sizes() = [2048, 1, 3, 3], strides() = [9, 1, 3, 1]
(3)INFO: Reducer buckets have been rebuilt in this iteration
Have you ever encountered such problems?
My results are 10% off from yours on the validation set. I suspect it was the warning above that caused the bad results.

**Comment by rulixiang** [comment]
Hey, I also got these infos and warnings. I tried to avoid them but sadly didn't succeed. So I believe they are not the reason for the performance gap. In my experiments, the variance of performance should be less than `~1%`. So I can't also understand why the performance drop is large in your experiments. I'll try to run the code again to check this repo, but no guarantee on time. Hope this comment could help you.
